Dockerfile

```diff
@@ -69,7 +69,7 @@ RUN make ${MAKE_TARGET}
 ####################################################################################################
 # Kubectl plugin image
 ####################################################################################################
-FROM scratch as kubectl-argo-rollouts
+FROM ubuntu:20.10 as kubectl-argo-rollouts
```

Comment:
Could you use the fully qualified name `docker.io/library/ubuntu:20.10` instead? This makes it more compatible in environments which do not assume dockerhub